test/second_q/drivers/test_driver.py

```diff
@@ -175,7 +175,7 @@ def test_driver_result_electronic_dipole(self):
             dipole = cast(ElectronicDipoleMoment, dipole)
 
             with self.subTest("x axis"):
-                mo_x_dip_ints = dipole.get_property("DipoleMomentX").get_electronic_integral(
+                mo_x_dip_ints = dipole._dipole_axes["DipoleMomentX"].get_electronic_integral(
```

Comment:
Although we are now accessing a private variable in all of these tests, I am willing to let this pass since this is going to be changed again very soon.